### PR TITLE
Refactor route-traffic command

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
Refactor route-traffic command

### CODE REVIEW

File: components/cli/pkg/commands/instance/route_traffic.go [hunk 1]
```diff
@@ -22,79 +22,84 @@ import (
...
+		return nil
+	}); err != nil {
+		return err
+	}
+	util.PrintSuccessMessage(fmt.Sprintf("Successfully routed %d%% of traffic to instance %s", percentage,
+		targetInstance))
+	return nil
+}
+
+func BuildRouteArtifact(cli cli.Cli, sourceInstances []string, dependencyInstance string, targetInstance string, percentage int,
```
Comment (isurulucky) [inline components/cli/pkg/commands/instance/route_traffic.go]: Do we need to make this function accessible from outside?

File: components/cli/pkg/commands/instance/testdata/virtual-services/pet-fe-src-vs.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+{
```
Comment (isurulucky) [inline components/cli/pkg/commands/instance/testdata/virtual-services/pet-fe-src-vs.yaml]: Why we need to have a json virtual service? If its required, should change the extension to json.

File: components/cli/pkg/commands/instance/route_traffic.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/commands/instance/route_traffic.go]: Updated.

File: components/cli/pkg/commands/instance/testdata/virtual-services/pet-fe-src-vs.yaml (hunk 2 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/commands/instance/testdata/virtual-services/pet-fe-src-vs.yaml]: Updated to json.

**Pull request merged**